demos/object_detection_demo_ssd_async/detection_model.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+// Copyright (C) 2018-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+#include "detection_model.h"
+#include <samples/args_helper.hpp>
+#include <samples/slog.hpp>
+
+using namespace InferenceEngine;
+
+DetectionModel::DetectionModel(std::string modelFileName, float confidenceThreshold, bool useAutoResize, const std::vector<std::string>& labels)
+:ModelBase(modelFileName){
+    this->useAutoResize = useAutoResize;
+    this->confidenceThreshold = confidenceThreshold;
+    this->labels = labels;
```

Comment:
Use initializer list instead.

Reply:
ok

Reply:
There are other places where initializer list should be used.